ganga/GangaLHCb/__init__.py

```diff
@@ -136,7 +136,7 @@ def postBootstrapHook():
     configDirac.setSessionValue('noInputDataBannedSites', [])
     configDirac.setSessionValue('RequireDefaultSE', False)
     configDirac.setSessionValue('proxyInitCmd', 'lhcb-proxy-init')
-    configDirac.setSessionValue('proxyInfoCmd', 'lhcb-proxy-info')
+    configDirac.setSessionValue('proxyInfoCmd', 'dirac-proxy-info')
```

Comment:
why this?

Reply:
LHCbDIRAC doesn't provide a function called `lhcb-proxy-info` so 'lhcb-proxy-info' is actually coming from the normal LbEnv environment causing DIRAC to be set up twice, i.e. equivalent to `lb-run LHCbDIRAC/prod lb-run LHCbDIRAC/prod exec dirac-proxy-info`.

I noticed as the new `lb-dirac` command resets the environment before sourcing DIRAC so it fails with `lhcb-proxy-info: command not found`.

Reply:
Should `lhcb-proxy-init` be changed in a similar way?

Reply:
Also, you could avoid sourcing LbEnv by depending on [`LbDiracWrappers`](https://pypi.org/project/LbDiracWrappers/) in the LHCb extra in `setup.py`.

Reply:
> Should lhcb-proxy-init be changed in a similar way?

No, LHCbDIRAC provides a (very slightly) customised version of `dirac-proxy-init`: https://gitlab.cern.ch/lhcb-dirac/LHCbDIRAC/-/blob/master/LHCbDIRAC/Core/scripts/lhcb-proxy-init.py